config.js

```diff
@@ -0,0 +1,20 @@
+/*
+  
+This file will be in the codebase and hence public. Add any passwords, 
+session keys and API keys to the .env file.  
+
+*/
+
+
+exports.keystoneInit = {
```

Comment:
why do you need `.keystoneInit` here? why not just export the object as is?

Reply:
`config.js` may contain other objects later?